tests/db/envelop_test.py

```diff
@@ -0,0 +1,13 @@
+# encoding=utf-8
+
+import unittest
+
+from alot.db.envelope import Envelope
+
+
+class TestEnvelopeMethods(unittest.TestCase):
+
+    def test_setitem(self):
+        e = Envelope()
+        e['Subject'] = u'sm\xf8rebr\xf8d'
```

Comment:
it's probably worhtleavinga comment about the bizare unicode string, otherwise everyone who reads this is going to wonder if that's something speicial, or if subject can be set to anything.